src/java/ognl/security/OgnlPolicy.java

```diff
@@ -0,0 +1,56 @@
+package ognl.security;
+
+import java.lang.reflect.ReflectPermission;
+import java.security.*;
+import java.util.Enumeration;
+
+/**
+ * Wraps current policy with minimum permissions plus user demand further permissions
+ * @author Yasser Zamani
+ * @since 3.1.23
+ */
+public class OgnlPolicy extends Policy {
```

Comment:
These two fields could be made final (IDE suggested).

Reply:
Deleted due to not being meaningful enough. Will be added in future if user demand.